pages/__404.md

```diff
@@ -5,4 +5,9 @@ changed: "2018-05-06"
 
 # Error 404 - Page not found.
 
-**Oh noes there's nothing in here.**
+Please use the menu or the search to find the content you have been looking for.
+
+Maybe you clicked a link somewhere on the web, pointing to a legacy (moved or deleted) «Accessibility Developer Guide» page.
+
+If you came here by clicking on a link within the «Accessibility Developer Guide», please contact us [Access4all/adg (GitHub.com)](https://github.com/Access4all/adg) and tell us, what link you clicked (and on which page).
```

Comment:
Could we add the e-mail here instead? I guess a developer would open an issue on Github anyway, while a non-developer will not even have a Github account. What do you think?

Reply:
Good suggestion. Done.